providers/aws/aws_provider.py

```diff
@@ -153,15 +193,14 @@ def provider_set_session(
         logger.info("Audit session is the original one")
         current_audit_info.audit_session = current_audit_info.original_session
 
-
+    logger.info("Checking if organizations metadata is needed ...")
```

Comment:
Are we going to use the three dots at the end of the logs?

Reply:
This logger line is not the same as line 129? I think it should not be here 
What do you think?